src/codeu/chat/client/commandline/Chat.java

```diff
@@ -238,17 +280,17 @@ public void invoke(Scanner args) {
     //
     panel.register("c-add", new Panel.Command() {
       @Override
-      public void invoke(Scanner args) {
-        final String name = args.hasNext() ? args.nextLine().trim() : "";
-        if (name.length() > 0) {
+      public void invoke(List<String> args) {
+        if(args.size() != 1){
+          System.out.println("ERROR: Failed to add new user");
```

Comment:
@mbassiouny Can you clarify this comment? Expecting the title of what, the conversation? Would something like "Failed to title new conversation" work or something else?